src/components/Footer.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+enum Filter {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
```

Comment:
Already you have enum in another component, why do you duplicate enum? Move enum to the separate file and use it everywhere 
<img width="335" alt="image" src="https://github.com/user-attachments/assets/577e04c4-1203-40f4-a3fc-e64a5f635f01" />